studio/components/ui/PageTelemetry.tsx

```diff
@@ -56,6 +56,12 @@ const PageTelemetry: FC = ({ children }) => {
       })
 
       post(`${API_URL}/telemetry/pageview`, {
+        ...(router.route.includes('/project/') && {
```

Comment:
@joshenlim @alaister 

guessing should be using app.selectedOrg.id | app.selectedProject.ref ?

sure you don't want to use org slug ? @pamelachia ?

Reply:
ahh yes can try something like this

```
const { slug, ref } = useParams()
```

to get either the project ref or org slug
if you need the org id specifically, then you might need to check against the list or orgs
```
const { data:organizations } = useOrganizationsQuery()
```

Reply:
I wonder if you could just use params from `useParams()` instead of breaking up the URL?

Reply:
thanks for calling this out. I think I made an error here. Will fix everything first 👍